src/ensembl/package.json

```diff
@@ -13,8 +13,9 @@
   "author": "EMBL-EBI",
   "license": "MIT",
   "scripts": {
-    "start": "npm run serve:dev",
-    "serve:dev": "webpack-dev-server --config ./webpack/webpack.config.dev.js",
+    "copy-dotenv": "test ! -f .env && (cp .env.example .env; echo '.env file created') || true",
+    "start": "npm install --no-save && npm run serve:dev",
+    "serve:dev": "npm run copy-dotenv && webpack-dev-server --config ./webpack/webpack.config.dev.js",
```

Comment:
Also, not sure whether `npm run copy-dotenv` command belongs here or in `npm start`

Reply:
Ideally, prod environment will have environment variables provided to it, which will be accessible to Node via the `process.env` field at build time.

Right now, if the variables are not provided, the defaults from `.env.example` will be used. If they are exist in the environment, they will override the defaults.

_(For example, NODE_ENV should already be changing to production — I guess via some webpack magic. At least it does so on my machine)._

Reply:
Thanks, @azangru. 

@imran-ebi, if you could have a quick look on this PR, we can get this merged.

Reply:
It would be great to get this on master as soon as practical as we're already losing quite a lot of time to mucking up commits due to different environments. If it's done in the next couple of days I'll cherry-pick it onto vizbi too. If not, it can wait till we merge/PR/squash-commit the vizbi branch.

Reply:
Just seen this has already been done! Sorry for the noise, :-) .